README.Mozilla.md

```diff
@@ -0,0 +1,85 @@
+Quick startup:
+==============
+
+All work will be done in ~/DeepSpeech/. Please update accordingly.
+This has been tested successfully on:
+* Ubuntu 14.04 with TITAN X GPU
+* Ubuntu 16.04 with K1000M GPU 
+* Debian Sid (as of september/october 2016) with GTX 970 GPU
+
+If you use different versions of CUDA/CUDNN please update accordingly below,
+CUDA version 8.0 and CUDNN version 5.1 are assumed.
+
+Compute capabilities should cover most usecase. If not, please refer to NVIDIA
+website: https://developer.nvidia.com/cuda-gpus and update the list with your
+needs.
+
+CUDA/cudnn path should be normalized (i.e., no "../" component) otherwise build
+system will not be happy.
+
+You might need an account on NVIDIA's developper area to download cudnn
+
+# System setup
+
```

Comment:
Might be worth mentioning that you need JDK8 and Swig for the build, and maybe mention the GCC-4.9 caveat here with the rest of the dependencies.